test/system/authorizing_profile_update_test.rb

```diff
@@ -1,7 +1,7 @@
 require "application_system_test_case"
 require "test_helper"
 
-class ProfileTest < ApplicationSystemTestCase
```

Comment:
When running tests locally, I receive the below error: ![Screenshot 2024-12-13 at 21 56 42](https://github.com/user-attachments/assets/f591d9ec-c9ac-4961-9b93-c19d478a772f)

I realize this is a bug I created in a previous PR since I used the same class name. Changing the file to another name allowed the tests to run.

Reply:
Moving this to its own PR https://github.com/rubygems/rubygems.org/pull/5366